craco.config.js

```diff
@@ -1,15 +1,34 @@
 const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
   webpack: {
-    alias: {
-      react: path.resolve('./node_modules/react'),
-      'react-dom': path.resolve('./node_modules/react-dom'),
-      'react-router': path.resolve('./node_modules/react-router'),
-      'react-router-dom': path.resolve('./node_modules/react-router-dom'),
-      '@zer0-os/zos-component-library': path.resolve('./node_modules/@zer0-os/zos-component-library'),
-    },
+    alias: {},
     configure: {
+      resolve: {
+        // https://webpack.js.org/configuration/resolve/#resolvefallback
+        // https://web3auth.io/docs/troubleshooting/webpack-issues#react-create-react-app
+        // https://docs.cloud.coinbase.com/wallet-sdk/docs/web3-react
+        fallback: {
+          crypto: false,
+          fs: false,
+          buffer: require.resolve('buffer'),
+          stream: require.resolve('stream-browserify'),
+          util: require.resolve('util/'),
+          assert: require.resolve('assert/'),
```

Comment:
Is there a reason to have these fallbacks? I tried a build with these all set to `false` and I haven't found any issues yet. What issues were you running into?

Reply:
The current web3 login needs them to function correctly. The app builds without it but when you try to open the wallet logins (specifically wallet connect or coinbase options), they error out.

Ideally libraries shouldn't need these fallbacks as it is considered bad practice and instead should use browser native functionality. I'd recommend later updating or changing the web3 login libraries and should be able to remove the fallbacks.

Reply:
So the other ones work without the polyfills?

Why don't we need this too? It seems to be pointed out in multiple places but we only added `Buffer`.

```
new webpack.ProvidePlugin({
	// Make a global `process` variable that points to the `process` package,
	// because the `util` package expects there to be a global variable named `process`.
       // Thanks to https://stackoverflow.com/a/65018686/14239942
	process: 'process/browser'
   })
```

Reply:
It appears to work without it - from my testing the `ProvidePlugin` is needed for WalletConnect but not Coinbase.

The other wallet integrations seem to be working fine but they error out for other configuration errors that seem unrelated (can try current staging environment to compare). The polyfills fundamentally are needed for `web3-react` in general so think it applies to all connectors.

Reply:
Alright, we ended up removing all options aside from `Metamask` and `Coinbase` so that limits our test maintenance area (considering most of the others didn't work at all).